src/test/java/uk/gov/cshr/service/InviteServiceTest.java

```diff
@@ -63,14 +64,21 @@ public void inviteCodesLessThan24HrsShouldNotBeExpired() {
                 .thenReturn(invite);
 
         MatcherAssert.assertThat(inviteService.isCodeExpired(code), equalTo(true));
+    }
 
-        ArgumentCaptor<Invite> inviteArgumentCaptor = ArgumentCaptor.forClass(Invite.class);
+    @Test
+    public void inviteCodesOldThanThreeDaysShouldBeExpired() {
```

Comment:
Could you please rename this method to inviteCodesOlderThanThreeDaysShouldBeExpired

Reply:
DONE.